### PR TITLE
AppsFlyer iOS v2 device mode updates

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Added iOS v2 device mode implementation steps.

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

## Notion ticket link

> [AppsFlyer device mode updates](https://www.notion.so/rudderstacks/AppsFlyer-iOS-v2-device-mode-updates-2b0dffc9964949bab6bfb703796a1769)

### CODE REVIEW

**Review by 1abhishekpandey — APPROVED** [review]

File: docs/destinations/attribution/appsflyer.mdx [hunk 1]
```diff
@@ -91,13 +91,14 @@ This setting is not applicable for the JavaScript (web) source as RudderStack do
 
 - Finally, click on **Next** to complete the configuration. AppsFlyer will be added and enabled as a destination in RudderStack.
 
-## Adding AppsFlyer to your project
+## Adding device mode integration
 
 Depending on your integration platform, follow the steps to add AppsFlyer to your project:
```
Comment (ishachopra0401) [inline docs/destinations/attribution/appsflyer.mdx:96]: Once you add AppsFlyer as a destination in the [RudderStack dashboard](https://app.rudderstack.com/), follow these steps to add it to your project depending on your integration platform:

File: docs/destinations/attribution/appsflyer.mdx [hunk 2]
```diff
@@ -154,6 +155,61 @@ RSConfigBuilder *builder = [[RSConfigBuilder alloc] init];
 </li>
 </ol>
       </TabPanel>
+      <TabPanel>
+        <div class="warningBlock">
+          This device mode integration is supported for AppsFlyer v6.5.4 and above.
+        </div>
+        Follow these steps to add AppsFlyer to your iOS project:
+<ol>
+<li>Add AppsFlyer as a destination in the <a href="https://app.rudderstack.com">RudderStack dashboard</a>.</li>
```
Comment (ishachopra0401) [inline docs/destinations/attribution/appsflyer.mdx:164]: Can be removed from here as moved above.

**Review by ishachopra0401 — APPROVED** [review]

**Review by pallabmaiti — APPROVED** [review]

File: docs/destinations/attribution/appsflyer.mdx (hunk 1 above)
Comment (ameypv-rudder) [inline docs/destinations/attribution/appsflyer.mdx:96]:
```suggestion
Once you add AppsFlyer as a destination in the [RudderStack dashboard](https://app.rudderstack.com/), follow these steps to add it to your project, depending on your integration platform:
```

File: docs/destinations/attribution/appsflyer.mdx (hunk 2 above)
Comment (ameypv-rudder) [inline docs/destinations/attribution/appsflyer.mdx:164]:
```suggestion
```

**Pull request merged**